src/main/java/com/epam/task/third/creator/CubeValidator.java

```diff
@@ -0,0 +1,93 @@
+package com.epam.task.third.creator;
+
+import com.epam.task.third.entity.Cube;
+import com.epam.task.third.entity.Point;
+import com.epam.task.third.logic.PointLogic;
+
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class CubeValidator {
+
+    private static final String CUBE_STRING_PATTERN = "(\\d+\\s\\d+\\s\\d+,){7}(\\d+\\s\\d+\\s\\d+\\n?)";
+    private final int countConnectedPointExpected = 3;
+    private PointLogic pointLogic;
+
+    public CubeValidator() {
+        pointLogic = new PointLogic();
+    }
+
+    public boolean validateString(String string) {
+        Pattern linePattern = Pattern.compile(CUBE_STRING_PATTERN);
+        Matcher matcher = linePattern.matcher(string);
+        return matcher.matches();
+    }
+
+    public boolean isCube(Cube cube) {
```

Comment:
1. it should be in different class
2. Do not create invalid cube, validate points before cube creation